recipes/whatlies/meta.yaml

```diff
@@ -0,0 +1,357 @@
+{% set use_all_subpackages = True %}
+
+{% set use_umap_packages = False %}
+{% set use_rasa_packages = False %}
+{% set use_fasttext_packages = False %}
+{% set use_spacy_packages = False %}
+{% set use_s2v_packages = False %}
+{% set use_tf_packages = False %}
+{% set use_transformers_dep = False %}
+{% set use_sentence_tfm_dep = True %}
+
+#######################################################
+
+package:
+  name: {{ name_ }}-meta
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 0b97c2b5b0bbc5c4044278ea8b2d3222cbb125e1fbf7bd854bb7f09b557e13d7
+
+build:
+  number: {{ build_num }}
+
+outputs:
+  - name: {{ name_ }}
+    build:
+      noarch: python
+    script: build_base.sh
+    requirements:
+      host:
+        - pip
+        - python {{ pyver }}
+      run:
+        - python {{ pyver }}
+        # base_packages
+        - altair {{ libver_altair }}
+        - bpemb {{ libver_bpemb }}
+        - gensim {{ libver_gensim }}
+        - matplotlib-base {{ libver_matplotlib_base }}
+        - scikit-learn {{ libver_scikit_learn }}
+      run_constrained:
+        # umap_packages
+        - umap-learn {{ libver_umap_learn }}
+        # rasa_packages
+        - rasa {{ libver_rasa }}
+        # fasttext_packages
+        - fasttext {{ libver_fasttext }}
+        # spacy_packages
+        - spacy {{ libver_spacy }}
+        - spacy-lookups-data {{ libver_spacy_lookups_data }}
+        # s2v_packages (+= spacy_packages)
+        - sense2vec {{ libver_sense2vec }}
+        # tf_packages
+        - tensorflow {{ libver_tensorflow }}
+        - tensorflow-text {{ libver_tensorflow_text }}
+        - tensorflow-hub {{ libver_tensorflow_hub }}
+        # transformers_dep
+        - transformers {{ libver_transformers }}
+        # sentence_tfm_dep
+        - sentence-transformers {{ libver_sentence_transformers }}
+    test:
+      imports:
+        - {{ name_ }}
+      commands:
+        - pip check
+      requires:
+        - pip
+
+  ### umap_packages
+  {% if use_umap_packages is defined %}
+  {% if use_umap_packages or use_all_subpackages %}
+  {% with subpackage_id = "umap" %}
+  - name: {{ name_ }}-{{ subpackage_id }}
+    build:
+      noarch: generic
+      string: {{ build_num }}_{{ subpackage_id }}_extra
+    requirements:
+      run:
+        # RECIPE: 
+        # "umap": base_packages + umap_packages
+        - {{ name_ }} {{ version }}
+        - umap-learn {{ libver_umap_learn }}
+    test:
+      imports:
+        - {{ name_ }}
+      commands:
+        - pip check
+      requires:
+        - pip
+  {% endwith %}
+  {% endif %}
+  {% endif %}
+  
+  ### rasa_packages
+  {% if use_rasa_packages is defined %}
+  {% if use_rasa_packages or use_all_subpackages %}
+  {% with subpackage_id = "rasa" %}
+  - name: {{ name_ }}-{{ subpackage_id }}
+    build:
+      ## NOTE: rasa is not available on conda-forge
+      skip: True  # [win or unix or osx]
```

Comment:
This is not possible, you're skipping all platforms. In this case, you'll have to remove this extra package for now or add rasa yourself.

Reply:
I started working on adding rasa. I am leaving that part of the script there for now and will use it once rasa is available.

Reply:
Ok but you cannot use `skip` in a noarch package. Then you‘ll have to use your switches. 
If you’re adding it already, why not add it to this PR?

Reply:
Thank you. Removed it.

Reply:
Not including rasa build at the moment. Will turn on rasa-subpackage build once it is ready.